test/basic.js

```diff
@@ -765,22 +765,33 @@ test('basic - catch apply throws', async t => {
 })
 
 test('basic - uncaught apply exception', async t => {
-  t.plan(1)
+  t.plan(2)
+
+  const [store] = await createStores(1, t)
 
   const error = new Promise((resolve, reject) => {
     process.on('uncaughtException', reject)
   })
 
-  const { bases } = await create(1, t, {
+  const a = new Autobase(store.session(), null, {
     async apply (nodes, view, base) {
       throw new Error('Synthetic')
-    }
+    },
+    valueEncoding: 'json'
   })
 
-  bases[0].append('trigger')
+  a.append('trigger')
```

Comment:
Sanity check (not related to this PR): no need to await this?

Reply:
not in this case because we want to catch the exception below